content/news/2025-08-02-qml-project.md

```diff
@@ -0,0 +1,73 @@
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_overlays_playlist.svg)
+
+2. **Enhanced Support for Lightweight Platforms**: The new design will be optimized for tablets, smartphones, and touchscreen laptops, ensuring a seamless experience across all devices.
+
+3. **Improved Search Capabilities**: The new search experience will offer better usability of advanced search options, making it easier to find and manage your tracks.
+
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_overlays_search.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_overlays_search3.svg)
+
+4. **Interactive Settings and Preferences**: We are introducing more interactive settings for audio routing, controllers, and waveforms, providing a more dynamic and user-friendly experience.
+
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_setting_audio_routing_advanced.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_setting_controller_overview.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_setting_controller_pointnclick.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_setting_waveform.svg)
+
+
+## Get Involved
+
+We are calling on developers and testers to join us in this exciting journey. Whether you are a seasoned developer or a passionate DJ looking to contribute, there are many ways you can help us.
+
+This significant project offers a fantastic opportunity to make a substantial impact from the ground up. By helping implement the new QML interface, fixing bugs, and adding features, you can shape the future of a widely-used DJ software. Your Qt/QML expertise will be crucial in this transition.
+Whether you're beefing up your portfolio or just want to give back to the open-source community, contributing to Mixxx is a great way to truly make a difference!
+
+
+## For Developers
+
+- **Code Contributions**: Help us implement the new QML interface, fix bugs, and add new features. Your expertise in Qt/QML will be invaluable as we transition to this new technology.
+  > **Pro tip**
+  >
+  > Contributing to Mixxx enhances your coding skills and connects you with a vibrant community, providing real-world impact and career opportunities. Many contributors have advanced their careers through Mixxx, gaining valuable experience and recognition. Join us to collaborate with passionate developers and open doors to exciting tech industry opportunities.
+- **Review and Feedback**: Review our code, provide feedback, and suggest improvements. Collaboration is key to our success, and your insights will help shape the future of Mixxx.

```

Comment:
should be emphasize that this is currently our bottleneck?

Reply:
Yeah, it's a good point, but hopefully this doesn't happen. If it doesn't we can always prepare some wiki to point these "reviews" to what a good review is supposed to be.
Ideally, we wouldn't want to discourage new contributor too soon